fix-yaml/__main__.py

```diff
@@ -0,0 +1,99 @@
+import json
+import logging
+import subprocess
+import sys
+from typing import List
+
+import yaml
+from github import Github, GithubException
+from pydantic import BaseSettings, SecretStr
+
+
+class Settings(BaseSettings):
+    input_token: SecretStr
+    input_target_repository: str
+    input_file_path: str
+    input_target_branch: str
+    input_commit_message: str
+    input_key_val_pairs: str
+
+
+logging.basicConfig(level=logging.INFO)
+settings = Settings()
+
+logging.info("Setting up GitHub Actions git user")
+subprocess.run(["git", "config", "user.name", "github-actions"], check=True)
+subprocess.run(
+    ["git", "config", "user.email", "github-actions@github.com"],
+    check=True,
+)
```

Comment:
```suggestion
subprocess.run(["git", "config", "user.name", "github-actions"], check=True)
subprocess.run(
    ["git", "config", "user.email", "github-actions@github.com"],
    check=True,
)
```

이 부분 역시 기본값을 포함하여 변수로 받아주면 좋을 것 같습니다.